plugins/org.jkiss.dbeaver.ext.postgresql/src/org/jkiss/dbeaver/ext/postgresql/PostgreUtils.java

```diff
@@ -616,13 +616,14 @@ public static void setArrayParameter(JDBCPreparedStatement dbStat, int index, Li
         }
     }
 
-    public static String getViewDDL(DBRProgressMonitor monitor, PostgreViewBase view, String definition) throws DBException {
+    public static String getViewDDL(DBRProgressMonitor monitor, PostgreViewBase view, String definition,
+                                    Map<String, Object> options) throws DBException {
```

Comment:
```suggestion
    public static String getViewDDL(
        DBRProgressMonitor monitor,
        PostgreViewBase view,
        String definition,
        Map<String, Object> options
    ) throws DBException {
```
You might also want to specify nullability annotations here.